pkg/api/navlinks/navlinks.go

```diff
@@ -0,0 +1,16 @@
+package navlinks
+
+import "github.com/grafana/grafana/pkg/api/dtos"
+
+func GetServerAdminNode(children []*dtos.NavLink) *dtos.NavLink {
+	return &dtos.NavLink{
+		Text:         "Server Admin",
+		SubTitle:     "Manage all users and orgs",
+		HideFromTabs: true,
+		Id:           "admin",
+		Icon:         "shield",
+		Url:          children[0].Url,
```

Comment:
In case this is used differently in the future, should we double check children length? Or is it overkill?

Reply:
Good question. I can't really think of situations where you'd want to add the `server admin` link, but have no children under it. But better safe than sorry, so I'll add a check for empty list.